edx_reverification_block/xblock/reverification_block.py

```diff
@@ -38,16 +38,15 @@ class ReverificationBlock(XBlock):
         display_name="Verification Attempts",
         default=0,
         scope=Scope.settings,
-        help="This is the number of attempts that students are permitted to "
-             "get a valid re-verification."
+        help="The number of times learners can attempt to verify their identity."
     )
 
     related_assessment = String(
-        display_name="Related Assessment",
+        display_name="Associated Assessment",
         scope=Scope.content,
         default=CHECKPOINT_NAME,
-        help="This name will allow you to distinguish distinct checkpoints "
-             "that show up in the reporting about student verification status."
+        help="The display name of the associated assessment. "
+             "You can create the assessment after you create the associated checkpoint."
```

Comment:
You can create the assessment either before or after you create the associated checkpoint.